### PR TITLE
Quartz - only notify QuartzSchedulerImpl.destroy() if the bean exists

### PR DESCRIPTION
**Pull request by mkouba**
- removes the warning mentioned in the #32566

### CODE REVIEW

**Review by gsmet — APPROVED** [review]
Nice trick, I'll keep it in mind!

File: extensions/quartz/runtime/src/main/java/io/quarkus/quartz/runtime/QuartzSchedulerImpl.java [hunk 1]
```diff
@@ -466,7 +467,7 @@ void start(@Observes @Priority(Interceptor.Priority.PLATFORM_BEFORE) StartupEven
      *
      * @param event ignored
      */
-    void destroy(@Observes @BeforeDestroyed(ApplicationScoped.class) Object event) {
+    void destroy(@Observes(notifyObserver = Reception.IF_EXISTS) @BeforeDestroyed(ApplicationScoped.class) Object event) {
```
Comment (punkratz312) [inline extensions/quartz/runtime/src/main/java/io/quarkus/quartz/runtime/QuartzSchedulerImpl.java:470]: styling: static import would be appreciate the code and make it leaner and therefor more explizit, by hiding whats not mandatory to tell and exist.

Comment (punkratz312) [reply extensions/quartz/runtime/src/main/java/io/quarkus/quartz/runtime/QuartzSchedulerImpl.java:470]: thanks for the mr, hopefully we get 3.0.0 fully released soon 👍

**Review by Pankraz76 — COMMENTED** [review]
thanks for the mr, hopefully we get 3.0.0 fully released soon 👍

**Pull request merged**